python/lsst/sconsUtils/tests.py

```diff
@@ -191,3 +195,68 @@ def run(self, fileGlob):
             self._env.Clean(target, self._tmpDir)
 
         return targets
+
+    def runPythonTests(self, pyList):
+        """Add a single target for testing all python files. pyList is
+        a list of nodes corresponding to python test files. The
+        IgnoreList is respected when scanning for entries. If pyList
+        is None, we will use automated test discovery within pytest.
```

Comment:
an empty list means auto pytest discovery. None means old-style discovery of *.py files. I agree that the distinction is not clear in this because I added to the documentation but did not change the original.